py/objint_mpz.c

```diff
@@ -411,6 +411,21 @@ mp_int_t mp_obj_int_get_checked(mp_const_obj_t self_in) {
     }
 }
 
+mp_uint_t mp_obj_int_get_uint_checked(mp_const_obj_t self_in) {
+    if (mp_obj_is_small_int(self_in)) {
+        return MP_OBJ_SMALL_INT_VALUE(self_in);
+    } else {
+        const mp_obj_int_t *self = MP_OBJ_TO_PTR(self_in);
+        mp_uint_t value;
+        if (mpz_as_uint_checked(&self->mpz, &value)) {
+            return value;
+        } else {
+            // overflow
```

Comment:
That seems quite superfluous, given the next line

Reply:
Hmm, right. Well, I just took it from `mp_obj_int_get_checked`. Will remove.

Reply:
Not present in new function.